src/store.ts

```diff
@@ -22,6 +22,7 @@ type UnwrapContainers<Containers extends Array<ValueContainer<unknown>>> = {
 
 export class Store {
   registry: Map<ValueContainer<Any>, string>;
+  containers: Map<string, ValueContainer<Any>>;
```

Comment:
naming got confusing between these two

Reply:
seems registry is containerNamesByRef and containers is caontainerRefsByName or something but  maybe we can simplify both and use one registry format?

Reply:
Or rename registry to namespaces?

Reply:
yeah, sounds good